week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/cart/cart-panel/cart-panel.component.ts

```diff
@@ -0,0 +1,20 @@
+import { article, p } from '../../../framework/dom-creators';
+import { cartItem } from '../cart-item/cart-item.component';
+import { cartItemSum } from '../cart-item/cart-item-sum.component';
+import { cartPanelType } from '../cart.types';
```

Comment:
Skoro w `hero.component` trzymasz typ w środku `hero.component` - to tutaj też bym do tego tak podszedł.

Twoim wyjątkowym typem jest jedynie `cartItemType` - które powinno być w oddzielnym dedykowanym pliku i reprezentować model danych (i być napisane z dużej jako: `CartItem` - ten przyrostek `Type` w nazwie nie jest konieczny)